.github/workflows/ci.yml

```diff
@@ -43,3 +43,25 @@ jobs:
         docker push ${{ secrets.IMAGE_NAME }}:${GITHUB_REF/refs\/tags\//}
         docker tag ${{ secrets.IMAGE_NAME }}:smi ${{ secrets.IMAGE_NAME }}:smi-${GITHUB_REF/refs\/tags\//}
         docker push ${{ secrets.IMAGE_NAME }}:smi-${GITHUB_REF/refs\/tags\//}
+  helm:
+    name: Helm charts publish
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        if: github.event_name != 'pull_request' && (startsWith(github.ref, 'refs/tags/') || github.ref == 'refs/heads/master') && success()
```

Comment:
@kumarabd this is not required, the following is already checked at the root.

Reply:
@kushthedude  but this is a separate job step right? Different from `docker`